# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tqt/fix-paths.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- tqt.git/mkspecs/linux-g++/qmake.conf.vanilla	2025-05-13 21:44:58.713280107 +0200
+++ tqt.git/mkspecs/linux-g++/qmake.conf	2025-05-13 21:45:15.533349420 +0200
@@ -39,12 +39,12 @@
 
 QMAKE_INCDIR		=
 QMAKE_LIBDIR		=
-QMAKE_INCDIR_X11	= /usr/X11R6/include
-QMAKE_LIBDIR_X11	= /usr/X11R6/lib
+QMAKE_INCDIR_X11	= /usr/X11/include
+QMAKE_LIBDIR_X11	= /usr/X11/lib64
 QMAKE_INCDIR_TQT		= $(TQTDIR)/include
 QMAKE_LIBDIR_TQT		= $(TQTDIR)/lib
-QMAKE_INCDIR_OPENGL	= /usr/X11R6/include
-QMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib
+QMAKE_INCDIR_OPENGL	= /usr/X11/include
+QMAKE_LIBDIR_OPENGL	= /usr/X11/lib64
 
 QMAKE_LINK		= g++
 QMAKE_LINK_SHLIB	= g++
